lib/matplotlib/backends/web_backend/js/mpl.js

```diff
@@ -140,15 +150,17 @@ mpl.figure.prototype._init_canvas = function () {
 
     mpl.ratio = (window.devicePixelRatio || 1) / backingStore;
 
-    var rubberband = $('<canvas/>');
-    rubberband.attr(
+    var rubberband_canvas = (this.rubberband_canvas = document.createElement(
+        'canvas'
+    ));
+    rubberband_canvas.setAttribute(
         'style',
         'position: absolute; left: 0; top: 0; z-index: 1;'
     );
 
     var pass_mouse_events = true;
 
-    canvas_div.resizable({
+    $(canvas_div).resizable({
```

Comment:
Is this putting jquery back in?

Reply:
It's needed for the only usage of jQuery UI. That'll be removed in the last PR.

Reply:
I see, this gets taken back out in https://github.com/matplotlib/matplotlib/pull/17086/commits/b398c0d6b416e6501c47e34cc301e1aefeaab148